ui/app/store/actions.js

```diff
@@ -1709,7 +1709,7 @@ export function exportAccount (password, address) {
 
     log.debug(`background.submitPassword`)
```

Comment:
Nit: This message is outdated

```suggestion
    log.debug(`background.verifyPassword`)
```